code/modules/mob/living/simple_animal/hostile/beheaded_kamikaze.dm

```diff
@@ -0,0 +1,30 @@
+/mob/living/simple_animal/hostile/beheaded_kamikaze
+	name = "beheaded kamikaze"
+	desc = "You can't be serious."
+	icon_state = "clown" //placeholder
+	icon_living = "clown" //placeholder
+	speak_chance = 80
+	speak = list("AAAAAAAAAAAAAAAAA", "AAAAAAAAAAAAAAAAAAAAAA", "AAAAAAAA", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "AAAAAAAAAA")
+	emote_taunt = list("screams", "yells")
+	taunt_chance = 80
+	speed = 0
+	maxHealth = 7
+	health = 7
+	ignored_damage_types = list(BRUTE = 0, BURN = 0, TOX = 1, CLONE = 0, STAMINA = 1, OXY = 1) //Set 0 to receive that damage type, 1 to ignore
+	environment_smash = 1
+	vision_range = 7
+	aggro_vision_range = 7
+	idle_vision_range = 7
+
+/mob/living/simple_animal/hostile/beheaded_kamikaze/Move()
+	playsound(src, 'sound/voice/attackkamikaze.ogg', 80)
```

Comment:
Needs to not be pitched imo, the best part about Serious Sam kamikazes is the sound stacking.